src/main/java/com/flightstats/hub/metrics/StatsdReporter.java

```diff
@@ -83,9 +106,7 @@ public void time(String channel, String name, long start, long bytes, String...
     }
 
     public void mute() {
-        if (dataDogHandler != null) {
```

Comment:
It always scares me when random bits of code check for null values on injected members... 👍